css/style.css

```diff
@@ -0,0 +1,660 @@
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::after,
+blockquote::before,
+q::after,
+q::before {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/*
+  ========================================
+  Main-side
+  ========================================
+*/
+main {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
+/*
+  ========================================
+  Misc
+  ========================================
+*/
+
+.gen-flex {
+  display: flex;
+}
+
+/*
+  ========================================
+  Navigation Bar
+  ========================================
+*/
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 1px 1%;
+  border: solid 1px #e2e2e2;
+  font-size: 14.25px;
+  font-family: Arial, Helvetica, sans-serif;
+  position: fixed;
+  width: 98%;
+  top: 0;
+}
+
+.left-nav {
+  display: flex;
+  align-items: center;
+}
+
+button {
+  background-color: #326891;
+  border: 0 solid #e2e2e2;
+  color: white;
+  font-weight: 700;
+  border-radius: 3px;
+  padding: 9px;
+}
+
+.space-cosmos {
+  color: #000;
+  text-decoration-line: none;
+  font-weight: 900;
+}
+
+.space-cosmos :hover {
+  text-decoration-line: underline;
+}
+
+/*
+  ========================================
+  Header
+  ========================================
+*/
+
+h1 {
+  width: 600px;
+  margin: 6% auto 3%;
+  font-family:
+    'Domine',
+    serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 46px;
+  color: #121212;
+  font-style: italic;
+}
+
+.top-section > * {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.img-header {
+  width: 70%;
+  color: #666;
+  font-size: 15px;
+  margin-bottom: 4px;
+}
+
+.image-credit {
+  color: #888;
+  font-size: 80%;
+}
+
+figcaption {
+  width: 60%;
+  margin: 0 32% 0 22%;
+}
+
+.byline {
+  flex-direction: column;
+  width: 43%;
+  margin: 3% 28.5%;
+}
+
+.author {
+  font-weight: bold;
+  margin-bottom: 1%;
+}
+
+.share-line {
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+/*
+  ========================================
+  Aside-Editors's Picks
+  ========================================
+*/
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  width: 210px;
+  grid-gap: 5%;
+  place-items: center;
+}
+
+.aside-img img {
+  width: 100%;
+}
+
+.aside-text a {
+  color: #000;
+  text-decoration-line: none;
+  font-size: 0.95rem;
+  font-weight: 500;
+}
+
+aside {
+  float: right;
+  clear: both;
+  position: absolute;
+  top: 250%;
+  right: 5%;
+  z-index: -1;
+}
+
+aside > p {
+  margin-bottom: 15px;
+  font-weight: 600;
+}
+
+.second-article {
+  clear: both;
+}
+
+/*
+  ========================================
+  Main
+  ========================================
+*/
+
+.flex {
+  display: flex;
```

Comment:
Guys, your class names are not descriptive and In large or professional projects, it can lead you to a lot of confusion. Try to change the name of this class. You already have the .gen-flex and this one is .flex (but it also align-items at the center). When naming general classes, use a descriptive name of what it really does.